.github/workflows/unit-tests.yaml

```
@@ -0,0 +1,54 @@
+    setup-and-run:
+      runs-on: ubuntu-latest
+      timeout-minutes: 60
+  
+      steps:
+        - name: Checkout repository
+          uses: actions/checkout@v3
+          with:
+            fetch-depth: 0 # Fetch all history for comparison
+  
+        - name: Set up Docker
+          uses: docker/setup-buildx-action@v2
```

actionlint (1.7.7)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)